types/prettier/index.d.ts

```diff
@@ -324,6 +350,7 @@ export interface SupportOptionDefault {
 }
 
 export interface SupportOption {
+    name: SupportOptionName;
```

Comment:
or
```suggestion
    name: string;
```

Reply:
Which should I do?

Reply:
I think that `string` is better because any strings can be option name for external plugins.

Reply:
I see.

Reply:
done :)